blog-posts/step-function-learnings/step-function-logs-learning.md

```diff
@@ -0,0 +1,92 @@
+---
+published: false
+title: "Stop looking for your express state machine logs, they don't exist"
+description: 'Logs for express Step Function with the CDK'
+tags: logs, stepfunction, serverless, cdk
+series:
+canonical_url:
+---
+
+In this article you will learn how to add **logs in an express step function** with the CDK because it is not enabled by default. When you go from a standard step function with logs to an express one that gives you nothing.
+
+TL;DR
```

Comment:
The TL;DR should be a single phrase that explains the solution with the configuration that needs to be added.
Here you are explaining everything, before revealing the solution. This is misleading.
Moreover, you should put it as a header